### DISCUSSION TITLE
Validator responds inccorectly

### DISCUSSION THREAD

**Discussion by Rashmi-sb** [CMSgov/price-transparency-guide-validator#40] [General]
Has anyone used this validator, we tested it and we see some of required the JSON entities names are not validated against correct schema entities name
Eg : Billing_code_type(schema naming ) is not validated to BillingCodeType(incorrect naming) , even if we remove this whole element from the output JSON , the validator says Valid JSON .

**Comment by jlamb0423** [comment]
I just installed the tool (node.js, git, docker), followed the CMS instructions and executed against test files - and it did say valid JSON.  I was not able to open the test file itself using a JSON Viewer like Dadroit or Notepad++ with JSON plug-in to see content.  What did you use to open the file (in a structured format)?
